lib/endsWith/index.js

```diff
@@ -0,0 +1,5 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function endsWith(suffix, list) {
+  return suffix === list[list.length - 1]
```

Comment:
what if `list` is null or empty?

Reply:
if empty returns false
if null - throws Error

Do not understand whether we using not-throwing-errors politics? I could make checking:

```js
return list ? s === l[l.length-1] : false
```

Reply:
Shorter `return list && s === l[l.length-1]```

Reply:
fix it for dev mode